torch/csrc/autograd/FunctionsManual.cpp

```diff
@@ -1266,30 +1266,25 @@ Tensor infinitely_differentiable_logit_backward(
   }
 }
 
-Tensor kl_div_double_backward_grad_output(const Tensor & grad, const Tensor & input, const Tensor & target, int64_t reduction, bool log_target) {
-  auto result = kl_div_backward(grad, input, target, at::Reduction::None, log_target);
+Tensor kl_div_backward(const Tensor& grad_output, const Tensor& input, const Tensor& target, int64_t reduction, bool log_target) {
```

Comment:
Isn't this going to be a performance regression over the previous implementation because previously there were handwritten fused kernels for these operations?

Reply:
Potentially, although just for CUDA and for `log_target=False` as the TI was just on that branch. It should be faster on the CPU side though.

I would not worry too much about this, as, in that case, just 3 kernels are being called. I reckon that this is not such a big deal (?)